AFQ/dki.py

```diff
@@ -71,7 +71,7 @@ def fit_dki(data_files, bval_files, bvec_files, mask=None,
     names = ['FA', 'MD', 'AD', 'RD', 'MK', 'AK', 'RK', 'params']
 
     if out_dir is None:
-        out_dir = op.join(op.split(data_files[0])[0], 'dki')
+        out_dir = op.join(op.split(data_files)[0], 'dki')
```

Comment:
I might be missing something, but I think I am doing that here:
https://github.com/yeatmanlab/pyAFQ/blob/master/AFQ/utils/models.py#L28

On Sun, May 8, 2016 at 8:17 PM, Ben Cipollini <notifications@github.com>
wrote:

> In AFQ/dki.py
> <https://github.com/yeatmanlab/pyAFQ/pull/21#discussion_r62447012>:
>
> > @@ -71,7 +71,7 @@ def fit_dki(data_files, bval_files, bvec_files, mask=None,
> >      names = ['FA', 'MD', 'AD', 'RD', 'MK', 'AK', 'RK', 'params']
> >
> >      if out_dir is None:
> > -        out_dir = op.join(op.split(data_files[0])[0], 'dki')
> > +        out_dir = op.join(op.split(data_files)[0], 'dki')
>
> I think you should check if data_files is a string at the top, and make a
> list out of it. In that case, this would bea list comprehension.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/yeatmanlab/pyAFQ/pull/21/files/78fe87de46aa2ce9e4a85ba589ff3ab0ff0c874a#r62447012>
>